azure-pipelines/secure-development-tools.yml

```diff
@@ -56,21 +47,23 @@ steps:
   inputs:
     SourceFolder: '$(Build.ArtifactStagingDirectory)\symbols'
     Contents: |
-      *.dll
-      *.pdb
+      **\*.dll
+      **\*.pdb
     TargetFolder: $(Agent.TempDirectory)\APIScanFiles
   condition: ne(variables['DisableAPIScan'], 'true')
 
-# ### Run APIScan
-#- task: APIScan@2
-#  displayName: Run APIScan
-#  inputs:
-#    softwareFolder: $(Agent.TempDirectory)\APIScanFiles
-#    softwareName: 'VS-MEF'
-#    softwareVersionNum: '$(Build.BuildId)'
-#    isLargeApp: false
-#    toolVersion: '3.1.29231'
-#  condition: ne(variables['DisableAPIScan'], 'true')
+### Run APIScan
+- task: APIScan@2
+  displayName: Run APIScan
+  inputs:
+    softwareFolder: $(Agent.TempDirectory)\APIScanFiles
+    softwareName: 'VS-MEF'
+    softwareVersionNum: '$(Build.BuildId)'
+    isLargeApp: false
+    toolVersion: 'Latest'
+  condition: ne(variables['DisableAPIScan'], 'true')
+  env:
+    AzureServicesAuthConnectionString: runAs=App;AppId=$(ApiScanClientId);TenantId=$(ApiScanTenant);AppKey=$(ApiScanSecret)
 
  ### Create formatted HTML code analysis log file.
 - task: securedevelopmentteam.vss-secure-development-tools.build-task-report.SdtReport@1
```

Comment:
You can remove the first copy of tasks `SdtReport@1` and `PublishSecurityAnalysisLogs@2` in favor of this second one.
Add `condition: succeededOrFailed()` to these `SdtReport@1` and `PublishSecurityAnalysisLogs@2`.

Reply:
Fixed.